### PR TITLE
Implement new design for uploading/removing avatars

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes vector-im/riot-web#8456
Fixes vector-im/riot-web#11287

Based on new design:
image
image

Result:
image
image
image

### CODE REVIEW

**Comment by t3chguy** [comment]
Blocked on https://github.com/matrix-org/matrix-react-sdk/pull/3732 otherwise it not possible to interact with the ImageView atop the Settings

**Comment by t3chguy** [comment]
Needs design for no-avatar state in Room Profile Settings

File: res/css/views/dialogs/_RoomSettingsDialog.scss [hunk 1]
```diff
@@ -42,3 +42,11 @@ limitations under the License.
     padding-left: 40px;
     padding-right: 80px;
 }
+
+// show a different AvatarSetting placeholder for RoomProfileSettings which is basically a clone of ProfileSettings
+.mx_RoomSettingsDialog .mx_AvatarSetting_avatar .mx_AvatarSetting_avatarPlaceholder::before {
+    mask: url("$(res)/img/feather-customised/image.svg");
```
Comment (dbkr) [inline res/css/views/dialogs/_RoomSettingsDialog.scss]: `avatar_placeholder.svg` or something maybe?

Comment (t3chguy) [reply res/css/views/dialogs/_RoomSettingsDialog.scss]: Nooooooo

Comment (dbkr) [reply res/css/views/dialogs/_RoomSettingsDialog.scss]: Oh I see, it really is an image

**Pull request merged**